sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2583,6 +2583,27 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       }
     }
   }
+
+  test("SPARK-33474: Support TypeConstructed partition spec value") {
```

Comment:
Why did you place the test in `sql/hive`? Are your changes hive specific?

Reply:
> Why did you place the test in `sql/hive`? Are your changes hive specific?

No , moved to sql module